atom/browser/native_window_mac.mm

```diff
@@ -1205,6 +1205,63 @@ static bool FromV8(v8::Isolate* isolate, v8::Handle<v8::Value> val,
   return collectionBehavior & NSWindowCollectionBehaviorCanJoinAllSpaces;
 }
 
+void NativeWindowMac::SetVibrancy(const std::string& type) {
+  if (!(base::mac::IsOSMavericks() || base::mac::IsOSYosemiteOrLater())) return;
+
+  NSVisualEffectView *vview = (NSVisualEffectView *)vibrant_view_;
```

Comment:
Should be `NSVisualEffectView* vview` and `(NSVisualEffectView*)`, I don't think the linter runs on Objective-C files.